### ISSUE TITLE
A11y: HTML lang attribute on documentation site

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#812]
As a developer, ensure the `<html>` tag on the documentation site has a lang attribute.  The missing lang attribute is being flagged as an issue on accessibility scans.  

fix: 
```
<html lang="en">
...
</html>

> **1** TBD
